content/en/logs/guide/identify-and-exclude-noisy-logs.md

```diff
@@ -0,0 +1,115 @@
+5. Switch group by `service` to group by `status`. This generates a top statuses list for your service.
+6. Click on the first listed status and select `search for` in the populated menu. This adds your `status` facet to the search.
+
+**Note**: These steps are applicable to any high volume logging query to generate a top list. You can group by any facet, such as `host` or `client name` versus `service` or `status`.
+
+## 2. Identify noisy logs with log patterns
+Switch to the [patterns][2] view, located next to the graph view, to view the patterns associated with your service and status.
+
+{{< img src="logs/guide/patterns.png" alt="Log Patterns View" responsive="true" style="width:100%;">}}
+
+The patterns view is helpful when identifying and filtering noisy patterns. It shows the number of logs matching a pattern, split by service and status. Click on the first pattern to view a detailed log of events relating to your status. A contextual panel will populate with information about your noisiest status pattern.
+
+## 3. Exclude noisy logs with an exclusion filter
+
+The selected pattern contextual panel lists every instance (event) of a log pattern and creates a custom search query based on the selected pattern. Use this query in an exclusion filter to remove those logs from your index. Note that excluded logs are still available in the [live tail view][3] and can be [sent to log archives][4], or used to [generate metrics][5].
+
+**To create an exclusion filter for your top log pattern**:
+
+{{< img src="logs/guide/exclusion-filter.png" alt="Log Patterns View" responsive="true" style="width:100%;">}}
+
+1. Click on your top pattern from the pattern view list.
+2. Click the *View All* button in the top right corner to automatically generate the search query associated with this pattern.
+3. Select the `</>` option to the right of the search query and copy the search query.
+4. Navigate to the Configuration page under Logs in the main left hand navigation. Select indexes and click on your associated index. This will populate the option to add an exclusion filter.
+5. Select “Add an Exclusion Filter”.
+6. Input the filter name, define the exclusion query by pasting the search query you copied, and set an exclusion percentage.
+
+Exclusion filters can be disabled at any time by toggling the disable option to the right of the filter. They can also be modified and removed by hovering over the filter and selecting the edit or delete option.
+
+**Note**: If a log matches several exclusion filters, only the first exclusion filter rule is applied. A log is not sampled or excluded multiple times by different exclusion filters.
+
+## 4. Generate metrics to monitor excluded logs
+
+{{< img src="logs/guide/generate-metric.png" alt="Log Patterns View" responsive="true" style="width:75%;">}}
```

Comment:
This image should go after the instructions to create the log-based metric as the final result of the instructions